Classes/Common/AbstractDocument.php

```diff
@@ -590,7 +590,7 @@ public static function &getInstance(string $location, array $settings = [], bool
             $instance = new IiifManifest($pid, $location, $iiif);
         }
 
-        if (!is_null($instance)) {
+        if ($instance != null) {
```

Comment:
Please always use strict comparisons!

```suggestion
        if ($instance !== null) {
```